ci/deploy_app.sh

```diff
@@ -0,0 +1,97 @@
+EQ_FEEDBACK_LIMIT="${EQ_FEEDBACK_LIMIT:=10}"
+HTTP_KEEP_ALIVE="${HTTP_KEEP_ALIVE:=650}"
+EQ_GOOGLE_TAG_MANAGER_ID="${EQ_GOOGLE_TAG_MANAGER_ID:=}"
+EQ_GOOGLE_TAG_MANAGER_AUTH="${EQ_GOOGLE_TAG_MANAGER_AUTH:=}"
+CDN_ASSETS_PATH="${CDN_ASSETS_PATH:=/design-system}"
+CONFIRMATION_EMAIL_LIMIT="${CONFIRMATION_EMAIL_LIMIT:=10}"
+EQ_SUBMISSION_CONFIRMATION_BACKEND="${EQ_SUBMISSION_CONFIRMATION_BACKEND:=cloud-tasks}"
+
+
+gcloud beta run deploy eq-questionnaire-runner \
+    --project="${PROJECT_ID}" --region="${REGION}" --concurrency="${CONCURRENCY}" --min-instances="${MIN_INSTANCES}" --max-instances="${MAX_INSTANCES}" \
+    --port=5000 --cpu="${CPU}" --memory="${MEMORY}" \
+    --image="${DOCKER_REGISTRY}/eq-questionnaire-runner:${IMAGE_TAG}" --platform=managed --allow-unauthenticated \
+    --set-secrets EQ_REDIS_HOST="redis-host:latest" \
+    --set-secrets EQ_REDIS_PORT="redis-port:latest" \
+    --set-secrets "/keys/keys.yml"="keys:latest" \
+    --set-secrets "/secrets/secrets.yml"="secrets:latest" \
+    --set-env-vars EQ_STORAGE_BACKEND="${EQ_STORAGE_BACKEND}" \
+    --set-env-vars EQ_ENABLE_SECURE_SESSION_COOKIE="${EQ_ENABLE_SECURE_SESSION_COOKIE}" \
+    --set-env-vars EQ_SECRETS_FILE="${EQ_SECRETS_FILE}" \
+    --set-env-vars EQ_KEYS_FILE="${EQ_KEYS_FILE}" \
+    --set-env-vars EQ_RABBITMQ_ENABLED="${EQ_RABBITMQ_ENABLED}" \
+    --set-env-vars EQ_ENABLE_HTML_MINIFY="${EQ_ENABLE_HTML_MINIFY}" \
+    --set-env-vars EQ_RABBITMQ_HOST="${EQ_RABBITMQ_HOST}" \
+    --set-env-vars EQ_RABBITMQ_HOST_SECONDARY="${EQ_RABBITMQ_HOST_SECONDARY}" \
+    --set-env-vars EQ_QUESTIONNAIRE_STATE_TABLE_NAME="${EQ_QUESTIONNAIRE_STATE_TABLE_NAME}" \
+    --set-env-vars EQ_SESSION_TABLE_NAME="${EQ_SESSION_TABLE_NAME}" \
+    --set-env-vars EQ_USED_JTI_CLAIM_TABLE_NAME="${EQ_USED_JTI_CLAIM_TABLE_NAME}" \
+    --set-env-vars EQ_SUBMISSION_BACKEND="${EQ_SUBMISSION_BACKEND}" \
+    --set-env-vars EQ_FEEDBACK_BACKEND="${EQ_SUBMISSION_BACKEND}" \
+    --set-env-vars EQ_PUBLISHER_BACKEND="${EQ_PUBLISHER_BACKEND}" \
+    --set-env-vars EQ_FULFILMENT_TOPIC_ID="${EQ_FULFILMENT_TOPIC_ID}" \
+    --set-env-vars EQ_INDIVIDUAL_RESPONSE_LIMIT="${EQ_INDIVIDUAL_RESPONSE_LIMIT}" \
+    --set-env-vars EQ_INDIVIDUAL_RESPONSE_POSTAL_DEADLINE="${EQ_INDIVIDUAL_RESPONSE_POSTAL_DEADLINE}" \
+    --set-env-vars EQ_FEEDBACK_LIMIT="${EQ_FEEDBACK_LIMIT}" \
+    --set-env-vars WEB_SERVER_TYPE="${WEB_SERVER_TYPE}" \
+    --set-env-vars WEB_SERVER_WORKERS="${WEB_SERVER_WORKERS}" \
+    --set-env-vars WEB_SERVER_THREADS="${WEB_SERVER_THREADS}" \
+    --set-env-vars WEB_SERVER_UWSGI_ASYNC_CORES="${WEB_SERVER_UWSGI_ASYNC_CORES}" \
+    --set-env-vars DATASTORE_USE_GRPC="${DATASTORE_USE_GRPC}" \
+    --set-env-vars HTTP_KEEP_ALIVE="${HTTP_KEEP_ALIVE}" \
+    --set-env-vars EQ_GCS_SUBMISSION_BUCKET_ID="${PROJECT_ID}-submission" \
+    --set-env-vars EQ_GCS_FEEDBACK_BUCKET_ID="${PROJECT_ID}-feedback" \
```

Comment:
These values should be set in the first block of variables and referenced here as it's a bit hidden here in the list of env vars

Reply:
Moved to first block of variables now!